unittest/CMakeLists.txt

```diff
@@ -1,24 +1,24 @@
-set(
-  source_files
-  TestUtil.cpp
-  main.cpp
-  test_Args.cpp
-  test_AtomicFile.cpp
-  test_Checksum.cpp
-  test_Compression.cpp
-  test_Config.cpp
-  test_Counters.cpp
-  test_FormatNonstdStringView.cpp
-  test_Hash.cpp
-  test_Lockfile.cpp
-  test_NullCompression.cpp
-  test_Stat.cpp
-  test_Statistics.cpp
-  test_Util.cpp
-  test_ZstdCompression.cpp
-  test_argprocessing.cpp
-  test_compopt.cpp
-  test_hashutil.cpp)
+set(source_files
+    TestUtil.cpp
+    main.cpp
+    test_Args.cpp
+    test_AtomicFile.cpp
+    test_ccache.cpp
+    test_Checksum.cpp
+    test_Compression.cpp
+    test_Config.cpp
+    test_Counters.cpp
+    test_FormatNonstdStringView.cpp
+    test_Hash.cpp
+    test_Lockfile.cpp
+    test_NullCompression.cpp
+    test_Stat.cpp
+    test_Statistics.cpp
+    test_Util.cpp
+    test_ZstdCompression.cpp
+    test_argprocessing.cpp
+    test_compopt.cpp
+    test_hashutil.cpp)
```

Comment:
That's because I have cmake-format installed and it reformats on save... I added test_ccache.cpp here. It aligned everything to the same indentation.

If you mind I can revert this part.

Reply:
Ah, I should have guessed that it was due to cmake-format but I didn't. Yes, I would appreciate not using cmake-format as mentioned in bbb6006913f321bc388c9f584f9d1848b44efcfa.